docs/source/concept_models/vivarium_ciff_sam/vivarium_conceptmodeldiagram_SAM.drawio

```diff
@@ -0,0 +1 @@
Yk8mtCRaa9AZ1+ZS9YhCKPpXrKM7Zo0RZD/AsPeyYUCX3Gb0rDBBP9KOq3rB//0huzxJuuKGHqelbeqZXpbPaJUd56KaZxpCsV6sALwmpC0CcSSRFFYhGBuL0QsQs7ar2oG/ETM35ogxJe50vB6YHJOk6zPMD8lknMbvjt7eXr+kCatIF+NQzpr/U0gUcySmsyvcDm+QK7NFTjxPO5xSZBnH7cin7EgGMITWjEWAnVBl7YwmgrfVjwPuiDKOZ6tObciLas7vgtEdBrqm9+MlPc5BBo7j8zmBFXm8wRi/UZoyciE5N5AlnWSGod0ROmbtdgtDSGiGnrdI8KqON5X8qnOJqteZeLbeNxKvKu4r42yLxDrDjestmYgs/FRAD3npDekdTcMQn4yJHjeE3Qhq/W4p0CaUYblJz0uKsRjaso9ex953Y2dVHHPtkKVS5upEkMs1m2MwqJnZkk5eiXIeoIstH2ljT4E6BTi/4L9BuI+3dIMYWfaPZDhxuH1mccwzx5SHK91FvQG9Q7rMX6AXGQFVXGBu7wdVRrPcMQLsrAAfWYyEvFE1j7DiWoRU/sBsc+XodhMAYcRqyakD2ayVpml4HmG7CfZK3K5wPE5Z7IQeagrpxEvVx0Aodi4/uGpBLg2sKUKKwCHNZ3FyqAdrO7toLUAA4gMK9AJWBrQtqewao8zLxiTQRU6ArPh2Dn0t3OENRNT57CIeeJz7tl4pPU8CU3tFthRwyl16qCRoXVdMdOOZ1B8Vw1UVL6/OK2DPYXV7Q3MO82YMbECQllHkLfEE7PYTukoAWahO8cH/6ZPXkdTTb3BA9hvwk3B4452jdCHKOHyjzfpsS77eycJ0ulrx+8T3COcpV79kLVCTenC6HILD5TQ1QOTRUTFWXWYN9RMLu17SbCuWxRvZtHGVlPjjz/LxGVPuNqDL5XYuoarJUO7b/+4eRTEXuo2bs0lunGYZqhWLkMnTnGWr4qrGtBWMvDUCNQ6sTm57vR4VcqS1zkJMG8S1VmOtD7ZXqqMg2W/tJj+Pe2qvxso3YPqA7rH8LIsRJNcTJqubRNiC4ypyxVvk5rrpryAILfYjGe5egL09kpqIsDgmB/GRRCMXzEILKRJrN13MiC9TSGpkNXyvflog3PpLbXwcTMdutwnFv4cauRy79hCi6r8UJu7lt6Zx0GbyMiFWA7zq8yIZb0ZeBH/7Y5q1pd9jIjzsd6K2Pu56PLSYk9wfYTyvzemQ6XG0/0rsGBk2dn8rhplJ9UjXoUbUdk+0QcSr84+O6AGgd2Yc47pVJ+cfinmjJS9xq5K8/83HFsSb4zf5F38QzWu9NeU+F9a/kpxE8acCQGDtsa1Qxqevb4XfYYdFHhLTHkD08slFjNvXjy7NLBhMuOt92wLC5SZobNfxUOt8vU7V42e7hOcxR+HsUjvCTF6wTSmmoTYgsStIZxtNK/vyLaEAgdqyzdFHNlHas01U1iTH2doHqyLYbP/EiWuOwzupmzolrpqx1F5R0guFjcv1xrf/+TDkLvvgpMcKp04HeM8Uz3/Nx6L2sRh8jyB/Rti2Wph13nzFpPEzvnrNgK9AQz1UocpWN1biqKqxhyiyiPvbhuzCh7ZQ8f0qPPKp+p5kYDXGqpo3WWTnvhHhEqZwNGAIzRfurL6wQci79NM0cu9oqxlnvoExZIjYcYTWlzYywMHtzHrrZfY9kXa/h1PZYAvoJYGm71nZYzODGj3OzftO/muFJbMB5NlGEkwqlcqqL4UhMBAbB44Btu7J5aL/XDFm5CS0UQtd0ll8NaOrLoQWg2bas8F7StscwFQENsMjrcH4sSXWAYe1xZh0l079p/OPEMlWh44w3NSiEuLwDhG+W2rguxeBdZ/yzP1R3I9IaREOO6XTVDf0EgMrs3JcHVLsW3ufhZaOuQLX59mh8zFg5UAcv6BtriKt2NU5Cqprbiv0a1nEPFjhAtj5G5gatJp9/p3NndPM4Al8QbljK5Ors/iF6B37/+S5e3n2b3/zn6vv3/456KLPerk2JitzpVqpW0SullNEQq4Y85W848PJg7dx9jm+5D5VVSksZIGs3phKqvSb1qKqzOgyqcoE8HFQdLv3D4ntetujxArmpeNVXMVq3ux97seJzf2PeHz73OuJ1mhVk/eOH3qsFr8KC58BpA5mrCEqz7vt4csyurd47zK7qAWTaf3EZcRVBr9BS5fQ2ZaE25MihtUPOHvRg3D7K7I+u7gm2RX85sVbDUxUMnOXEHXxCW7ymZyjnsCwfHdz/ASqFn7JyonOHn3YxJPxKr3KpeMGOGdiAna8bqahMhZMisF0bvfNHYOP2DeeDQN4esctOJUfCoPSpci1BqBwYTfvOmMO68QDvwXDsrsDg+3Y40Dk2MHpwh5yVdGoKQmdQDJ6LdiZ9Xl5b+B3dw3YC8Bv6cAR80xONBazaIpDXzvhEBdUIHDzONpgA3OVwPHUAiun9TlcfL+BjvZa6pAQ5BLvaqAn5YikbngSR9+OPhR/KYhXM0YXYp279gN3x13q5qni4isWyGC71tlHfXVH6nRNV/maXbSCrLJcGVqquPc7nZuqWaVu1PdXlqQoKpbq4p6RAQAMLdSR/BtRGOego4oXsyIa9aFs/XcoRHgsBuA17WI9xKdckT5dqfYKcQohbHf4ZQU78UOED01rH5qFiyYuhrORfjsjeYtxDIXJf9OQUsDpsM1FOZena4MDgJ+KfzqUaqj0YgDugynLYDuqK8uLF6qC+irNBai9a+kH5ROhQUA2FAfFRpqD7s0u5mY7sL+ilLfcJep7Mpp4nY1jPJ7KNMf+wFYuvomoemUE788eJXXlkfzzrN9R7Js57N5y7cRSRGzyCmYtKbSktRL1gWWAfrr7+8rk5Qs5Nd/m6tXDLYhuilvaly3NzeuhvIEWbpGl7e+2rythjZMb2KAl3bsGGKdpDSUL+0UGm01GxkiS58t+ms9Ajl0TmpNXbiWRY3Gd9y9Cb/wM=</diagram></mxfile>
```

Comment:
I don't think this file needs to go in the repo. Do let me know if I'm wrong about that, though.